frame/benchmarking/src/lib.rs

```diff
@@ -193,6 +193,7 @@ macro_rules! benchmarks {
 		$( $rest:tt )*
```

Comment:
I think the doc should be improved with the new functionality.

Also the manual tests should probably be removed, and `extra` and `skip_meta` attribute should be explained but this is not part of this PR.

Reply:
The `skip_meta` was not obvious to me at all, doc would be much appreciated.

Reply:
actually I don't know either for `skip_meta`, but we can document them in another PR, this PR just need to include the documentation for the new feature: i.e. document that we can write impl_benchmark_test_suite inside the benchmark macro directly in order to get a split test suite with potential better error message.

Reply:
Okay done @thiolliere

Reply:
thanks